tests/scripts/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Test the condition helper."""
```

Comment:
Move tests for the `__init__` to `test_init.py` but keep `__init__.py` around as an empty file.